catalog/core/catalog-core-directorymonitor/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -38,10 +38,14 @@
                     label="Monitor in place" value="in_place"/>
         </AD>
 
+        <AD description="Optional: Blacklisted file pattens.  These are regular expressions used to exclude different file patterns from being ingested.  The default values are temporary Windows files (*.tmp) and Windows thumbnails (thumbs.db)."
+            name="Blacklisted File Patterns" id="blacklistedFilePatterns" required="false" type="String"
+            cardinality="100"
+            default=".*.tmp,thumbs.db"/>
```

Comment:
According to the documentation, "matching is case in-sensitive from Camel 2.17 onward".  We currently use 2.18, so this shouldn't be an issue.  http://camel.apache.org/file2.html